DOC-3914 ECS task scale docs (#4420)
harness/developer-hub: docs/chaos-engineering/technical-reference/chaos-faults/aws/ecs-task-scale.md

        </tr>
    </table>
    <h3>Optional tunables</h3>
    <table>
      <tr>
        <th> Variables </th>
        <th> Description </th>
        <th> Notes </th>
      </tr>
      <tr>
        <td> TOTAL_CHAOS_DURATION </td>
        <td> Duration that you specify, through which chaos is injected into the target resource (in seconds). </td>
        <td> Default: 30 s. For more information, go to <a href="../common-tunables-for-all-faults#duration-of-the-chaos"> duration of the chaos.</a> </td>
      </tr>
      <tr>
        <td> SEQUENCE </td>
        <td> Sequence of chaos execution for multiple services. </td>
        <td> Default: parallel. Supports parallel and serial sequences. For example, 30 s. For more information, go to <a href="../common-tunables-for-all-faults#sequence-of-chaos-execution">sequence of chaos execution. </a></td>
      </tr>
      <tr>
        <td> RAMP_TIME </td>
        <td> Period to wait before and after injecting chaos (in seconds).</td>
        <td> For example, 30 s. For more information, go to <a href="../common-tunables-for-all-faults#ramp-time"> ramp time.</a> </td>
      </tr>
    </table>

### ECS service name

> Comma-separated tasks that are a part of the particular service, that are scaled. Tune it by using the `SERVICE_NAMES` environment variable. 

Reviewer: ```suggestion
Comma-separated service name that derives the task. Tune it by using the `SERVICE_NAMES` environment variable.
```

The following YAML snippet illustrates the use of this environment variable:

[embedmd]:# (./static/manifests/ecs-task-scale/task-service-name.yaml yaml)
```yaml
# scale the tasks of an ECS cluster
apiVersion: litmuschaos.io/v1alpha1
kind: ChaosEngine
metadata:
  name: engine-nginx
spec:
  engineState: "active"
  annotationCheck: "false"
  chaosServiceAccount: litmus-admin
  experiments:
  - name: ecs-task-scale
    spec:
      components:
        env:
        # provide the name of ECS cluster
        - name: CLUSTER_NAME
          value: 'demo'
        - name: SERVICE_NAMES
          value: 'test-svc'
```

### ECS task replicas

Number of replicas to scale up (or down) to. Tune it by using the `REPLICAS` environment variable.

The following YAML snippet illustrates the use of this environment variable:

[embedmd]:# (./static/manifests/ecs-task-scale/task-replicas.yaml yaml)
```yaml
# scale the tasks of an ECS cluster
apiVersion: litmuschaos.io/v1alpha1
kind: ChaosEngine
metadata:
  name: engine-nginx
spec:
  engineState: "active"
  annotationCheck: "false"
  chaosServiceAccount: litmus-admin
  experiments:
  - name: ecs-task-scale
    spec:
      components:
        env:

>         # provide the name of ECS cluster

Reviewer: we can check this comment
